internal/featuretests/v3/featuretests.go

```diff
@@ -160,15 +155,15 @@ func setup(t *testing.T, opts ...interface{}) (cache.ResourceEventHandler, *Cont
 		}()
 		return srv.Serve(l) // srv now owns l and will close l before returning
 	})
-	g.Add(eh.Start())
+	g.AddContext(eh.Start)
```

Comment:
this is the workgroup that would have to be dropped if we wanted to drop the entire workgroup package.

Reply:
https://github.com/projectcontour/contour/issues/4206